tests/foreman/cli/test_subnet.py

```diff
@@ -170,7 +170,7 @@ def test_negative_create_1(self):
 
     @run_only_on('sat')
     @tier1
-    def test_negative_create_2(self):
+    def test_negative_create_with_pool_range(self):
```

Comment:
I think for consistency sake test name should be `test_negative_create_with_address_pool`  since we already have similar `positive` test.